sled-agent/types/src/rack_init.rs

```diff
@@ -95,9 +95,10 @@ pub mod back_compat {
             })
         }
     }
-    impl From<RackInitializeRequestV1> for RackInitializeRequest {
+
+    impl From<RackInitializeRequestV1> for RackInitializeRequestV2 {
         fn from(v1: RackInitializeRequestV1) -> Self {
-            RackInitializeRequest {
+            RackInitializeRequestV2 {
```

Comment:
Just checking that I'm not missing something - we had to add `V2` just because of the `skip_timesync` addition, right? (I'm hoping we can drop it entirely if so.)